toil/src/cgcloud/toil/toil_box.py

```diff
@@ -0,0 +1,108 @@
+import logging
+from cgcloud.core import fabric_task
+from cgcloud.mesos.mesos_box import MesosBox, MesosMaster, MesosSlave
+from cgcloud.fabric.operations import sudo
+from bd2k.util.strings import interpolate as fmt
+from cgcloud.lib.util import abreviated_snake_case_class_name
+from cgcloud.core.common_iam_policies import ec2_full_policy, s3_full_policy, sdb_full_policy
+from fabric.context_managers import settings
+from textwrap import dedent
+
+log = logging.getLogger( __name__ )
+
+mesos_version = '0.22'
+
+user = 'toilbox'
+
+install_dir = '/opt/mesosbox'
+
+log_dir = '/var/log/mesosbox/'
+
+ephemeral_dir = '/mnt/ephemeral'
+
+persistent_dir = '/mnt/persistent'
+
+class ToilBox(MesosBox):
+    def __init__( self, ctx):
+        super( ToilBox, self ).__init__( ctx)
+        self.lazy_dirs=set()
+
+    def _pre_install_packages( self ):
+        super( ToilBox, self)._pre_install_packages()
+
+    def _post_install_packages( self ):
+        super( ToilBox, self )._post_install_packages( )
+        self._install_boto( )
```

Comment:
This will soon be obsolete when https://github.com/BD2KGenomics/toil/issues/207 is resolved. Maybe add a FIXME?